packages/components/src/components/tag/tag.stories.ts

```diff
@@ -29,173 +34,86 @@ export const Default = {
+ * Use the `disabled` attribute to disable a tag.
  *
- * If you add icons to the slot, please make sure to account for accessibility by providing an alt-text.
- */
-
-export const Slots = {
-  parameters: {
-    controls: { exclude: ['size', 'selected', 'removable', 'disabled'] }
-  },
-  render: (args: any) => {
-    return html`
-      ${['default', 'removable-indicator'].map(slot =>
-        generateTemplate({
-          axis: {
-            x: {
-              type: 'slot',
-              name: slot,
-              title: 'slot=...',
-              values: [
-                {
-                  title: slot,
-                  value:
-                    slot === 'default'
-                      ? `<div class="slot slot--background slot--border h-auto w-[108px] text-[8px]"></div>`
-                      : `<div slot="removable-indicator" class="slot slot--background slot--border h-auto w-4 text-[8px]"></div>`
-                }
-              ]
-            }
-          },
-          args,
-          constants: [
-            { type: 'attribute', name: 'removable', value: 'true' },
-            { type: 'attribute', name: 'size', value: 'lg' }
-          ]
-        })
-      )}
-    `;
-  }
-};
-
-/**
- * Use the `base`, `content` and `removable-indicator` part selectors to customize the button.
+ * __Hint:__ If the href attribute is set i.e. the tag is rendered as a link (`<a>`),
+ * the disabled attribute is ignored, as links may not be disabled.
+ * To disable the tag in this case the href attribute has to be removed.
  */
 
-export const Parts = {
-  parameters: {
-    controls: { exclude: ['size', 'selected', 'removable', 'disabled'] }
-  },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: {
-          type: 'template',
-          name: 'sd-tag::part(...){outline: solid 2px red}',
-          values: ['base', 'content', 'removable-indicator'].map(part => {
-            return {
-              title: part,
-              value: `<style>#part-${part} sd-tag::part(${part}){outline: solid 2px red;} #part-${part} .${part}{outline: solid 2px red;}</style><div id='part-${part}'>%TEMPLATE%</div>`
-            };
-          })
-        }
-      },
-      constants: [
-        { type: 'template', name: 'width', value: '<div style="width: 300px">%TEMPLATE%</div>' },
-        {
-          type: 'slot',
-          name: 'default',
-          value: '<span class="content">Tag</span>'
-        },
-        {
-          type: 'slot',
-          name: 'removable-indicator',
-          value:
-            '<sd-icon slot="removable-indicator" library="system" name="close" label="remove" class="removable-indicator">'
-        },
-        { type: 'attribute', name: 'removable', value: 'true' },
-        { type: 'attribute', name: 'size', value: 'lg' }
-      ],
-      args
-    });
-  }
+export const Disabled = {
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-tag disabled>Disabled</sd-tag>
```

Comment:
https://github.com/solid-design-system/solid/pull/1226/files#r1709682861

Reply:
<img width="621" alt="image" src="https://github.com/user-attachments/assets/86558d83-75a8-4955-9769-68334c594684">

Reply:
Fixed in Figma. Label is now "Disabled"